__tests__/hooks/useLanguage.js

```diff
@@ -0,0 +1,26 @@
+import { timeouts } from '../constants.json';
+
+// selenium-webdriver API doc:
+// https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
+
+jest.setTimeout(timeouts.test);
+
+test('getter should return language set in props', async () => {
+  const { pageObjects } = await setupWebDriver({
+    props: {
+      locale: 'zh-YUE'
+    }
+  });
+
+  const [groupTimestamp] = await pageObjects.runHook('useLanguage');
+
+  expect(groupTimestamp).toMatchInlineSnapshot(`"zh-YUE"`);
+});
+
+test('getter should return default language if not set in props', async () => {
+  const { pageObjects } = await setupWebDriver();
+
+  const [groupTimestamp] = await pageObjects.runHook('useLanguage');
+
+  expect(groupTimestamp).toMatchInlineSnapshot(`"en-US"`);
+});
```

Comment:
Test to make sure there isn't a setter?

Reply:
Yap. I am changing all previous "setter test" to test for falsy, instead of test for exceptions.